### ISSUE TITLE
New definition molecular function regulator

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#20862]
Molecular function regulator regulates its target via non-covalent binding that does not result in covalent modification to the target. Examples of molecular function regulators include regulatory subunits of multimeric enzymes and channels. Mechanisms of regulation include allosteric changes in the target and competitive inhibition. 

Discussed on the go-tickets jamboree
@vanaukenk @thomaspd

**Comment by pgaudet** [comment]
Edited slightly to 
**A** molecular function regulator regulates **the activity of** its target via non-covalent binding that does not result in covalent modification to the target. Examples of molecular function regulators include regulatory subunits of multimeric enzymes and channels. Mechanisms of regulation include allosteric changes in the target and competitive inhibition.

**Comment by ValWood** [comment]
This works fine *except *. for "transcription regulators" which are protein modifiers 
for example:

some 
"General transcription initiation factors" are protein  kinases and ubiquitin ligases
some "transcription coactivators" are kinases  (mediator srb10)
some "transcription coregulators" are acetyltransferase

This goes back to an issue I have raised a couple of times on the transcription working group.
"transcription regulator" is a term biologists use, but it isn't describing a specific activity. It is really saying "this regulates the process of transcription in some way, we may not know precisely how". Similarly "GO:0140223   general transcription initiation factor activity" is a collection of disparate activities involved in the process of transcription initiation.

But I do agree with the description above for the description of a regulator, and it really holds out except for the transcription cases.

**Comment by pgaudet** [comment]
How about if we moved 'transcription coregulator' activity as a direct child of molecular_function (like we did for GTFs) - it would look like this 

<img width="419" alt="image" src="https://user-images.githubusercontent.com/4782928/107752498-20678100-6d1f-11eb-9b37-a8e73dbdcb2b.png">


@colinlog @RLovering thoughts ?

**Comment by colinlog** [comment]
This looks fine.

**Comment by ValWood** [comment]
In this case should there be a comment along the lines"
"Molecular function regulator regulates its target via non-covalent binding that does not result in covalent modification to the target." 
on all of the 
protein kinase regulator/activator/inhibitor terms?

Otherwise, it is totally non-obvious that you would not use:

"protein serine/threonine kinase inhibitor activity"
for a protein kinase, or phosphatase that inhibits a protein kinase. Many are even called inhibitors for example wee1 is the "
"M phase inhibitor protein kinase"


The current definitions do not exclude modifier inhibitors
"protein serine/threonine kinase inhibitor activity"
Binds to and stops, prevents or reduces the activity of a protein serine/threonine kinase.

**Comment by ValWood** [comment]
Although the numbers annotated to both. are relatively low:

<img width="1025" alt="Screenshot 2021-03-03 at 15 18 03" src="https://user-images.githubusercontent.com/7359272/109828179-50c28100-7c34-11eb-905d-b96ea1d1f71e.png">


and most for activator/kinase probably come from:
cyclin-dependent protein kinase activating kinase activity

**Comment by colinlog** [comment]
I was wondering whether GO:0003712 coTF (and indirectly its 2 co-activator and co-repressor children) should also be children of the process transcription regulation (GO:0006355), just as GO:0003700 dbTF currently is?

**Comment by pgaudet** [comment]
Added the missing relation @colinlog is asking. 

@ValWood I dont quite understand what you are asking.

**Comment by ValWood** [comment]
You propose to redefine "molecular function regulator " to exclude "regulation by protein modification" (i.e restrict to allosteric regulation /active site obstruction etc by a physically bound activator/inhibitor etc).

However, the protein kinase activator/inhibitor terms do not make this clear. People can (and have) used these terms to annotate upstream inhibitor and activator kinases. This is natural because these kinases are usually described as inhibitors and activators.   Also, terms in the ontology, for example, "cyclin-dependent protein kinase activating kinase activity" violate this rule because they are instantiated as both a 'kinase' and an 'enzyme activator'.

I propose 3  things:
1. remove the term "cyclin-dependent protein kinase activating kinase activity" (this might be in other tickets already), this is a mini GO -CAM because it described the activity and the substrate).

2. Update the definitions of 
GO:0019887   protein kinase regulator activity
Modulates the activity of a protein kinase, an enzyme which phosphorylates a protein.

and all the children to make it clear that these terms should not be used to annotate upstream kinases and phosphatases that are regulators, and a comment that these should be curated with a kinase/phosphatase term, and the substrate (and ideally a GO-Cam to capture the relationship (activation or inhibition).
(One of the keys for the GO use of 'enzyme activator'  seems to be that the regulator remains bound constitutively while it is activating- can the definition(s) make this much clearer in some way?) 

currently:
Molecular function regulator regulates its target via non-covalent binding that does not result in covalent modification to the target.

Isn't very easy to parse, and the definition focused on what a regulator doesn't do rather than what it does do.

3. Once 1. Is done the number of violations should decrease, but the remaining violations should be reported and corrected.

Sorry for expecting you to be a mind reader!

**Comment by pgaudet** [comment]
Thanks @ValWood I will do that.

WRT transcription factor and cofactors - I will put back coregulator under transcription regulator. Transcription regulator activity IS NOT a molecular function regulator, it is part_of regulation of transcription. I did this too fast before.

**Comment by ValWood** [comment]
I still don't like this definition because it describes a molecular function partially by what it isn't.

A molecular function regulator regulates the activity of its target via non-covalent binding that does not result in a covalent modification to the target. Examples of molecular function regulators include regulatory subunits of multimeric enzymes and channels. Mechanisms of regulation include allosteric changes in the target and competitive inhibition.

 I wonder if  the appropriate definition should be not that it does not result in covalent modification, but that : 
a 'molecular function regulator is constitutively  bound to its substrate  when its regulation activity is present (i.e. excludes transient binding and modification) '.  This covers  also for the exceptions where a regulator modifies it's substrate and remains bound to its substrate (I think this might be the case for some of the ubiquitin system regulators?)

**Comment by colinlog** [comment]
Hi Val, here my current thoughts on this:

(A) Your statement "I still don't like this definition because it describes
a molecular function partially by what it isn't." is an overstatement in my
opinion. The definition is clear as to what the regulator does do (binding
non-covalently to regulate ...) and the negative part is a way to exclude
the post-translational modifications. So it is not quite as bad as the
'poor man's' definition of epigenetics that says 'an activity that does not
change the DNA sequence' which is a purely negative definition that in fact
only describes what it is not, rather than what it is (a (de)methylase, a
(deacetylase) a (de)ubiquitinase, etc...).

What I wonder, is whether the enzyme activity regulation paradigm(s) (that
is what this ticket is about, isn't it) can be generalized to all
regulators? Genetics tells us that biosynthetic pathways and signal
transduction pathways are very different types of objects. For example, in
genetic epistasis analysis the upstream enzyme is epistatic to the enzymes
that require its product to do their reactions, whilst in signal
transduction pathways, it is the most downstream effector that will be
epistatic to its upstream regulators.

Hence, I fear that making 'enzyme regulation' 'the top term of regulation'
might impede continued development signal transduction pathways. Could some
wise people look at this? My gut feeling is that you have regulation of
activity (i) by physical interaction and/or (ii) by covalent modification.
(i) can be equivocated with a 'sequestration/recruitment' activity, either
to (ia) an ultrastructural CC (organelle, membrane-bound or not), or (ib)
in stable (in)active complex formation without subcellular localisation
modulation. (ii) is chemistry, and involves mechanisms of the type (i) to
achieve a longer lasting memory of an event than the binding event itself
was, which is then the basis for signal *transduction*, a signal that
travels through time and space*.*

If I got Pascale's point, as soon as multiple activities are involved it is
a biological process (BP), and then type (ii) is always a process (a
covalent protein modification function alters a different molecular
function) whilst type (i) is 'stand-alone' and incarnated by a protein
(complex) and therefore can be a molecular activity (MF).
Best, Colin

**Comment by vanaukenk** [comment]
In looking at inferred annotations in the context of https://github.com/geneontology/minerva/issues/437
I see that in Noctua we get inferred annotations to 'protein kinase regulator' terms based on the logical definition 'molecular_function' and regulates some 'protein kinase activity' (or a similarly defined child).

Based on what we're saying in this ticket, though, those annotations don't seem correct.

Do we instead want to define the 'protein kinase regulator' terms as 'molecular_function_regulator' and regulates some 'protein kinase activity'?

Here's the model:  http://noctua.geneontology.org/editor/graph/gomodel:6197061700002169?model_id=gomodel:6197061700002169

**Comment by ValWood** [comment]
OK, I  think the definition would be OK. However, this means that, for example, the kinase subunits of "mediator complex" would not be annotated as "transcription coactivator activity". As long as this is clear (and would make sense to biologists) I'm OK with that.

Similarly (I expect)  this would mean that the  kinase subunits of TFIIH and TFIIK would not be annotated as "general transcription initiation factor activity".
(this is a related issue, 2 ways to say the same thing, although this time not a MF regulator term).

However, I'm not sure how you get around the logic problem pointed out by @vanaukenk  in the comment above.
In Noctua, presumably a kinase that phosphorylates and regulates another kinase  would by inference be a molecular function regulator.  I don't know how you can prevent that?

**Comment by pgaudet** [comment]
This ticket #25165 addresses @vanaukenk 's comment above to *some* extent (but not completely). 


Closing because there is no action requested in the present ticket.

**Comment by colinlog** [comment]
I fail to understand the point Val? 

Specifically, 
Why would the mediator complex (including its cyclin and cyclin-dependent kinase subunit) not have co-activator activity anymore? Why would TFIIH subunits not be GTFs? You cannot be suggesting that they are not involved in transcription initiation? Any GO rule that prohibits TFIIH subunits to be GTFs is suspect to me. Is your opinion that these are good rules because they would mean that subunits of a complex do not have the activity of the complex anymore? Perhaps this ticket is then confounded by considering Mfs for individual proteins in isolation versus the emergent MF of a protein complex that requires each of the (intrinsic) MFs of its subunits? In that case, perhaps the activity of any protein complex is a BP, not a MF since more than one polypetide, each with its own MF is involved?

OR 

Is it about a wish to generally consider post-translational modifications as non-regulatory, but rather as signal transduction? This would be fine to me when we look at positive activities, those that enable the process, The dominoes that entrain the fall of other dominoes when they fall, the molecular causality chains that GO is capturing.

 However, when a process is stalled until one serine is (de)phosphorylated, that is 'true regulation' in the sense that the flux of the process (how much it 'does' iots process per time unit a.k.a. its rate/frequency of execution) is controlled via ***repression*** of the intrinsic activity (MF) of (at least one of) the proteins that 'do the job'. 

Generally, the smallest regulatory motif is when a repressor of the process is itself repressed (this also happens within one protein with multiple domains). That amounts to activation of the process. The strictest definition of regulators would be that true regulators are not part of the process they regulate, but their activity is necessary to allow the process to occur because the process-intrinsic components include a negative activity that stalls the process which must be counteracted for the process to take place. Then regulators are the repressors of those repressors. The hand that turns the water tap is a regulator: turning to the left or to the right will determine how much water flows out of the tap. Without this, the tap would shed water at a constant rate. (I think I explained a similar logic with traffic lights in another ticket one year ago). Cells have a lot of brakes that stop processes from running all the time. Regulating the brakes is how cellular homeostasis is achieved. Thismeans that for regulation to be possible there is a background of many MFs that are 'latently' active; awaiting the GO signal. The underlying concept is that proteins are metastable structures. They can exist in more than one 3D configuration, with active and inactive architectures for the same protein.

Perhaps two complementary and exclusive definitions of BP ***regulator*** must be used.
 (A) regulation of a process by "changing the cellular concentration of its protein components"  This must be via changing transcription, translation and/or degradation. This is why differential gene expression must be considered regulatory. It anticipates what the cell will be able to do.
 (B) "changing the output rate of the process without changing the cellular concentrations of the components of the BP"  

But this ticket is not about BPs, it is about regulatory MFs,isn't it. A regulatory MF would change the rate of a second MF. This can happen within one polypeptide or between two polypeptides, within one protein complex or between two protein complexes (eg Mediator and RNA polymerase II) and of course some scenarios have many more players...

Generalizations and the necessary simplifications they entail tend to obscure/not be able to encompass the complexity of molecular biology that GO should help to capture. For instance, phosphorylation (or methylation or acetylation etc.) can

(i) affect enzyme activity (enabling or disabling) via some substrate/product binding event or reconfiguration of catalytic residues. One well know example is the CDK tyrosine kinase WEE1 and  Y15 of CDC2 which affect its capacity to bind ATP when it is phosphorylated
(ii) create a binding site for a phosphoprotein binding domain, that can be part of the protein itself, or be present in a second protein, or remove such a binding site.

Indeed, after (ii) many molecular mechanisms can be at play, from destruction of the protein (following (A) above) to its sequestration/liberation [a.k.a. targeting, activation] following (B) above. But that should not obscure the fact that a regulatory MF needs a rate limiting activity to counteract, some repression that can be alleviated. This can be intrinsic to one polypeptide, or involve multiple polypeptides.

Perhaps what I try to say is that GO should not have to struggle with a kinase having another kinase as a substrate. It should be handled the same as when the kinase has a small  GTPase as substrate. Post-translational modification-based signal transduction motif is not exceptional. Mechanistically, there are many scenarios, but I think most will be type (i) the MF is changed or type (ii) more than one MF is involved , hence it is a BP that is regulated 

Is that the problem here? That BPs are being modelled as Mfs and MFs as BPs? Or is it that we struggle with the intrinsic activity of protein complexes and entire signal transduction pathways which we intuitively want to call MFs but are in fact BPs because multiple MFs (ormultiple instances of one type of MF)  are involved? Conversely, when a polypeptide has two domains, does it then have two MFs and therefore must be treated as having one intrinsic BP made up of those two MFs? 

Not sure this helps, but I did have an alarm bell going when you said these TFIIH subunits are not GTFs. That cannot be the outcome. GO can annotate the complexes, or GO can annotate protein complex functions to their subunits. Altogether  losing the MF GTF and coTF for these kinases but not for the other subunits of these complexes, simply because they are kinases sounds weird to me.
